ui/Dockerfile

```diff
@@ -16,6 +16,9 @@ RUN mv ./contracts/build ./ && rm -rf ./contracts/* ./contracts/.[!.]* && mv ./b
 COPY ./commons ./commons
 
 COPY ./ui ./ui
+ARG DOT_ENV_PATH=./ui/.env
```

Comment:
how did you check that it works?

Reply:
It’s tested by jobs in two ways:
1. The ultimate tests prove that it works with default path „ui/.env”.
The behaviour when deploying components is unchanged, because „ui/.env” is copied anyway in previous step.
2. Passing e2e tests prove that it works with overriden path to .env

Reply:
Thanks!